### DISCUSSION TITLE
How to add some lines or text on top of already rendered image?

### DISCUSSION THREAD

**Discussion by yk0n9** [emoon/rust_minifb#302] [Q&A]
How to add some lines or text on top of already rendered image. Is there such a related library?
I've rendered a picture and I want to overlay some text or lines on top of it.

**Comment by emoon** [comment]
Hey. There might be some crates that does support this, but I don't know them at the top of my head.

Not long ago I did write a very basic text render that I use for an private project. What I could do is to put that code into an example.

**Reply by yk0n9** [reply]
ok, i hope i can learn from it

**Reply by yk0n9** [reply]
thanks

**Reply by emoon** [reply]
np, I hope to have done later today.

**Reply by emoon** [reply]
![text_example](https://user-images.githubusercontent.com/350822/199101915-352c9224-e0a2-4a6a-9379-b990d440f3d2.png)

Plasma + text example added here: https://github.com/emoon/rust_minifb/blob/master/examples/text.rs

The text is very basic, but was good enough for what I needed

**Reply by yk0n9** [reply]
ok thanks❤
